tgui/packages/tgui-polyfill/ie8.js

```diff
@@ -458,7 +458,7 @@
         ) : (
           (parentNode = self.parentNode) /* && live(self) */ ?
             parentNode.dispatchEvent(e) :
-            true
+            null
```

Comment:
You should revert this bit because it's vendored code.

Reply:
Understood, was not really sure about this one.